### PR TITLE
Add youtube link for complete Kubernetes tutorial issue #1889

### LINKED ISSUE #1889: [Add] <Add youtube link for complete Kubernetes tutorial>
### Site URL

https://www.youtube.com/watch?v=y3WTwzx5ABk

### Description of the above provided site

Edureka's Kubernetes YouTube tutorial offers a comprehensive and beginner-friendly guide to understanding and mastering Kubernetes, a powerful container orchestration platform.

### Category

Youtube

### Sub Category

Kubernetes

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am participating in Hacktoberfest 2023

### PR DESCRIPTION
**Pull request by alienishi**
## Fixes Issue

Closes #1889 

## Changes proposed

Added YouTube link for complete Kubernetes tutorial

## Note to reviewers

I have added Edureka's YouTube tutorial link for Kubernetes, as mentioned in issue #1889 
@Anmol-Baranwal , @CBID2, and @rupali-codes, please review my PR, and if it is in accordance with guidelines, please do merge it :)

### CODE REVIEW

**Comment by alienishi** [comment]
@Anmol-Baranwal, @rupali-codes, and @CBID2, can you also please add hacktoberfest-accepted label, such that this contribution of mine will also get counted there :)

File: database/youtube/Kubernetes.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[
+    {
+        "name": "Edureka Kubernetes Tutorial",
+        "description": "Edureka's Kubernetes YouTube tutorial offers a comprehensive and beginner-friendly guide to understanding and mastering Kubernetes, a powerful container orchestration platform.",
```
Comment (CBID2) [inline database/youtube/Kubernetes.json:4]: [proofreading] You can make this a bit shorter. 
```suggestion
        "description": "Edureka's Kubernetes YouTube tutorial offers a comprehensive and beginner-friendly guide to understanding and mastering Kubernetes.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @alienishi! :) I pointed out a proofreading error in my inline comment. In the future, consider running your descriptions through Grammarly or Spellcheck before making a pull request. Other than that, you're off to a great start! :)

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

**Comment by alienishi** [comment]
Hi @CBID2 will surely keep in my mind your suggestions :)
Should I change it and create another Pull Request ?

**Comment by Anmol-Baranwal** [comment]
@alienishi 
Yeah, it will be approved and merged.

**Review by CBID2 — APPROVED** [review]
Looks good to me!

**Review by rupali-codes — APPROVED** [review]
lgtm

**Pull request merged**